src/adapter/sources.ts

```diff
@@ -902,13 +902,25 @@ export class SourceContainer {
 
     try {
       sourceMap.map = await this.sourceMapFactory.load(source.sourceMap.metadata);
-    } catch (e) {
-      this._dap.output({
-        output: sourceMapParseFailed(source.url, e.message).error.format + '\n',
-        category: 'stderr',
-      });
+    } catch (urlError) {
+      try {
```

Comment:
Can you add a comment to this block to say what's going on? Also wonder whether this is a code path we could enable only for VS.

Reply:
Done